test/e2e/ingress.go

```diff
@@ -596,7 +597,316 @@ var ____ = describe("Ingress tests custom routes", func() {
+	var (
+		cs  kubernetes.Interface
+		jig *ingress.TestJig
+	)
+
+	It("Should create path routes ingress v1 [Ingress]", func() {
+		jig = ingress.NewIngressTestJig(f.ClientSet)
+		cs = f.ClientSet
+		serviceName := "skipper-ingress-test-pr"
+		serviceName2 := "skipper-ingress-test-pr2"
+		ns := f.Namespace.Name
+		hostName := fmt.Sprintf("%s-%d.%s", serviceName, time.Now().UTC().Unix(), E2EHostedZone())
+		labels := map[string]string{
+			"app": serviceName,
+		}
+		labels2 := map[string]string{
+			"app": serviceName2,
+		}
+		port := 8080
+		replicas := int32(3)
+		targetPort := 9090
+		backendContent := "be-foo"
+		backendContent2 := "be-bar"
+		route := fmt.Sprintf(`* -> inlineContent("%s") -> <shunt>`, backendContent)
+		route2 := fmt.Sprintf(`* -> inlineContent("%s") -> <shunt>`, backendContent2)
+		waitTime := 10 * time.Minute
+
+		// CREATE setup
+		// backend deployment
+		By("Creating a deployment with " + serviceName + " in namespace " + ns)
+		depl := createSkipperBackendDeployment(serviceName, ns, route, labels, int32(targetPort), replicas)
+		_, err := cs.AppsV1().Deployments(ns).Create(context.TODO(), depl, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		By("Creating a 2nd deployment with " + serviceName2 + " in namespace " + ns)
+		depl2 := createSkipperBackendDeployment(serviceName2, ns, route2, labels2, int32(targetPort), replicas)
+		_, err = cs.AppsV1().Deployments(ns).Create(context.TODO(), depl2, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Creating service " + serviceName + " in namespace " + ns)
+		service := createServiceTypeClusterIP(serviceName, labels, port, targetPort)
+		_, err = cs.CoreV1().Services(ns).Create(context.TODO(), service, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Creating service " + serviceName2 + " in namespace " + ns)
+		service2 := createServiceTypeClusterIP(serviceName2, labels2, port, targetPort)
+		_, err = cs.CoreV1().Services(ns).Create(context.TODO(), service2, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Creating ingress " + serviceName + " in namespace " + ns + "with hostname " + hostName)
+		ing := createIngressV1(serviceName, hostName, ns, "/", netv1.PathTypePrefix, labels, nil, port)
+		ingressCreate, err := cs.NetworkingV1().Ingresses(ns).Create(context.TODO(), ing, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		addr, err := jig.WaitForIngressAddress(cs, ns, ingressCreate.Name, waitTime)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = cs.NetworkingV1().Ingresses(ns).Get(context.TODO(), ing.Name, metav1.GetOptions{ResourceVersion: "0"})
+		Expect(err).NotTo(HaveOccurred())
+
+		// skipper http -> https redirect
+		By("Waiting for skipper route to default redirect from http to https, to see that our ingress-controller and skipper works")
+		err = waitForResponse(addr, "http", waitTime, isRedirect, true)
+		Expect(err).NotTo(HaveOccurred())
+
+		// ALB ready
```

Comment:
If we do not set LB type on ingress the type would be default LB type which could be NLB

Reply:
yes true and we might want to convert the e2e tests in the future to default to NLB and use some specific tests for ALB.